conans/test/__init__.py

```diff
@@ -5,4 +5,4 @@
 os.environ["CONAN_RECIPE_LINTER"] = "False"
 
 # Enable warnings as errors only for `conan[s]` module
-warnings.filterwarnings("error", module="(.*\.)?conans\..*")
+# warnings.filterwarnings("error", module="(.*\.)?conans\..*")
```

Comment:
I'm commenting this out.  Now the helper `conan_v2_behavior` is raising warnings at many more places and we'd need to catch these warnings at several places in the code. I think it is better, from now on, to move these warnings to actual errors using `CONAN_V2_MODE`.